app/src/main/java/lapuk_app/views/main/ui/pages/PrivacyPolicy.kt

```diff
@@ -0,0 +1,138 @@
+package lapuk_app.views.main.ui.pages
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults.topAppBarColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import com.example.lapuk_app.R
+import lapuk_app.views.main.ui.theme.LapukTheme
+import lapuk_app.views.main.ui.elements.BottomBar
+import lapuk_app.views.main.ui.elements.TopBar
+
+@Preview(showBackground = true,
+    device = "spec:width=1080px,height=2400px,dpi=440,navigation=buttons"
+)
+@Composable
+fun PrivacyPolicyPagePreview() {
+    LapukTheme {
+        val navController = rememberNavController() // Create a mock NavController for preview
+        PrivacyPolicyPage(navController = navController)
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun PrivacyPolicyPage(navController: NavController) {
+    LapukTheme {
+        Scaffold(
+            topBar = {
+                TopBar()
+            },
+            content = { innerPadding ->
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(innerPadding)
+                        .padding(16.dp),
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    // title
+                    Box(modifier = Modifier.padding(vertical=18.dp), contentAlignment = Alignment.Center) {
+                        Text(
+                            modifier = Modifier
+                                .padding(12.dp),
+                            text = "PRIVACY POLICY",
+                            fontSize = 36.sp
```

Comment:
no need to individually declare font sizes, try fontStyle = Typography.titleLarge or something, basta naa nay titleLarge titleMedium tas titleSmall same with body text, naa tanan naka define sa Type.kt